core/domain/summary_services.py

```diff
@@ -222,3 +223,26 @@ def get_featured_exploration_summary_dicts(language_codes):
         reverse=True)
 
     return _get_displayable_exp_summary_dicts(sorted_exp_summaries)
+
+def get_top_rated_exploration_summary_dicts(language_codes):
+    """Returns a list of top rated explorations with the given language code.
+
+    The return value is sorted in decreasing order of search rank.
+    """
+    filtered_exp_summaries = [
+        exp_summary for exp_summary in
+        exp_services.get_non_private_exploration_summaries().values()
+        if exp_summary.language_code in language_codes]
+
+    search_ranks = {
+        exp_summary.id: exp_services.get_search_rank_from_exp_summary(
```

Comment:
employ average rating algorithm, I don't think we need to progress into too much as I have yet to employ the lower bound wilson score algorithm, but it is sorted based on number of scores followed by number of rankings.